src/app/pages/properties/add-to-wishlist/add-to-wishlist.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component} from '@angular/core';
+import { PropertiesWishlistService } from 'src/app/services/properties/properties-wishlist/properties-wishlist.service';
+import { ToastrService } from 'ngx-toastr';
+import { Router } from '@angular/router';
+
+
+@Component({
+  selector: 'app-add-to-wishlist',
+  templateUrl: './add-to-wishlist.component.html',
+  styleUrls: ['./add-to-wishlist.component.scss']
+})
+export class AddToWishlistComponent {
+
+
+  constructor(
+    private wishedPropertiesService: PropertiesWishlistService,
+    private router: Router,
+    private toastrService: ToastrService, ) {
+
+  }
+
+  addToWishlist(): void {
+    this.wishedPropertiesService.addToWishlist('house-at-he-Peak-of-the-mountain').subscribe(
```

Comment:
Hey @espoirMur , Kindly update this. We don't want users to add the same house to their wishlist over and over again.

Reply:
Done bro